<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbResolution">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>128x32</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>128x64</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btStart">
        <property name="text">
         <string>Start emulator</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="gbEmu">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Emulator</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="chkEnabled">
           <property name="text">
            <string>Enable</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbInverse">
           <property name="text">
            <string>Inverse</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Offset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbOffset">
           <property name="maximum">
            <number>128</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbFlipY">
           <property name="text">
            <string>Flip Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbFlipX">
           <property name="text">
            <string>Flip X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkRotate">
           <property name="text">
            <string>Rotate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Contrast</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slContrast">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>255</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Color scheme</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbColors">
           <property name="currentIndex">
            <number>2</number>
           </property>
           <item>
            <property name="text">
             <string>White</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blue</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blue+Yellow</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>slContrast</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
